Library/Homebrew/language/python.rb

```diff
@@ -153,7 +153,8 @@ def needs_python?(python)
       def virtualenv_install_with_resources(options = {})
         python = options[:using]
         if python.nil?
-          wanted = %w[python python@2 python2 python3 python@3 pypy pypy3].select { |py| needs_python?(py) }
+          pythons = %w[python python2 python3 pypy pypy3] + Formula.select { |f| f.name.start_with? "python@" }
```

Comment:
By some reason, this doesn't work as expected (but `%w[python python@2 python2 python3 python@3 python@3.8 pypy pypy3]` works)

Reply:
Whoops, of course it should be `Formula.map(&:name).select { |f| f.start_with? "python@" }` 👀
Fixed!